includes/api/ApiQueryRecentChanges.php

```diff
@@ -408,6 +410,10 @@ public function extractRowInfo( $row ) {
 				$vals['userid'] = $row->rc_user;
 			}
 
+			if ( $this->fld_useravatar ) {
+				$vals['useravatar'] = AvatarService::getAvatarUrl( $vals['user'], AvatarService::AVATAR_SIZE_MEDIUM );
```

Comment:
If someone sets the props as `userid|useravatar`, this code would be executed but `$vals['user']` would be undefined as the `user` prop wasn't set, so it's probably better to use `$row->rc_user_text` here.